src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioProjectTracker_IVsSolutionLoadEvents.cs

```diff
@@ -218,13 +218,23 @@ private void OutputToOutputWindow(string message)
             OutputToOutputWindow($"\tCreating '{projectName}':\t{commandLineArguments.SourceFiles.Length} source files,\t{commandLineArguments.MetadataReferences.Length} references.");
```

Comment:
No - this is [Conditional("DEBUG")] - it never shows up in customer builds, it's just convenient for debugging.